helm/fiftyone-teams-app/README.md.gotmpl

```diff
@@ -64,6 +64,7 @@ for steps on how to upgrade your delegated operators.
   - [Text Similarity](#text-similarity)
 - [Values](#values)
   - [Deploying On GKE](#deploying-on-gke)
+  - [`FIFTYONE_APP_BANNER_TEXT` and `FIFTYONE_APP_BANNER_COLOR`](#fiftyone_app_banner_text-and-fiftyone_app_banner_color)
```

Comment:
@findtopher sorry to over-pinging 🥴 

This is the magic you were talking about right? you add to gotmpl and when pre-commit runs, it makes README

Reply:
Hang tight